.github/workflows/ci.yml

```diff
@@ -56,7 +56,7 @@ jobs:
       - name: Build
         run: |
           # Download nydus components
-          NYDUS_VER=v$(curl -s "https://api.github.com/repos/dragonflyoss/image-service/releases/latest" | jq -r .tag_name | sed 's/^v//')
+          NYDUS_VER=v2.1.2
```

Comment:
Strange, the e2e test never got empty `NYDUS_VER` before. Is it really a github rate limitation problem? cc @liubin